### PR TITLE
Issue #13345: Enable AvoidStarImportCheckExamplesTest

### PR DESCRIPTION
**Pull request by TanayMorakhia**
Issue: #13345

### CODE REVIEW

**Comment by romani** [comment]
please read https://ci.appveyor.com/project/Checkstyle/checkstyle/builds/48223492/job/q68pt8d0lnb3oaow#L123 and resolve problem.

**Comment by romani** [comment]
https://github.com/checkstyle/checkstyle/blob/master/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstarimport/Example1.txt

you need to put under `// xdoc section -- end`
code
`class Example1 {}`

same for all other files.

**Comment by TanayMorakhia** [comment]
@romani Kindly review it. 
2 tests that have failed are due to inappropriate PR title, which I have changed.

**Review by romani — APPROVED** [review]
Ok to merge

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstarimport/Example1.java [hunk 1]
```diff
@@ -6,10 +6,14 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.imports.avoidstarimport;
+
 // xdoc section -- start
 import java.util.Scanner;         // OK
```
Comment (rnveach) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstarimport/Example1.java:12]: @romani I thought we were removing OKs?

**Review by rnveach — COMMENTED** [review]
No other issues.

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstarimport/Example1.java (hunk 1 above)
Comment (romani) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstarimport/Example1.java:12]: Yes, I missed this.
@TanayMorakhia , please remove ok comments that has no text in comment.

Comment (rnveach) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstarimport/Example1.java:12]: @romani Do we have OK removal check on xdoc examples? Is this just missing a suppression removal?

Comment (TanayMorakhia) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstarimport/Example1.java]: done

**Comment by romani** [comment]
Please add to your commit changes in xdoc files for such lines.

CI is complaining on this https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=17007&view=logs&j=c902ebb4-c9f8-5f09-4e17-ff78fbbc842e&t=9ca98c81-ff64-58f0-9d03-a23ac1c4a111&l=1760

**Comment by TanayMorakhia** [comment]
Sorry, missed it because my mvn also did not gave error on local machine.

**Comment by rnveach** [comment]
@romani https://github.com/checkstyle/checkstyle/pull/13846#discussion_r1355373837

**Comment by romani** [comment]
We need to remove suppression if there is some

**Comment by TanayMorakhia** [comment]
@romani any other changes needed ?

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by romani** [comment]
We do not have it for Examples, 
We hy it only for Inputs https://github.com/checkstyle/checkstyle/issues/13213

Right now there is very active resolving process for Examples activation, enforcing suppression will be Complicated and time consuming. I would rather wait for completion of active phase of updates and create suppression and issue to fix examples in scope of this issue.

**Review by rnveach — APPROVED** [review]

**Pull request merged**